src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.NtAuth.cs

```diff
@@ -112,6 +112,11 @@ private static async Task<HttpResponseMessage> SendWithNtAuthAsync(HttpRequestMe
                                 IPHostEntry result = await Dns.GetHostEntryAsync(authUri.IdnHost).ConfigureAwait(false);
                                 hostName = result.HostName;
                             }
+
+                            if (!isProxyAuth && !authUri.IsDefaultPort)
```

Comment:
Why not for proxy auth?

If it's just to minimize change, then we should add a comment about this. Actually we should probably add a comment regardless.

Reply:
yes, it was minimize the change. I was not sure but since nobody complained about proxies, I left it as it was.
I'm also not sure if the the proxy default port would match web server port.

Reply:
Ok. Can we add a comment?